noir_stdlib/src/string.nr

```diff
@@ -5,7 +5,7 @@ impl<N> str<N> {
     fn as_bytes(_self: Self) -> [u8; N] { }
```

Comment:
Why is this not marked public? Similarly with slice methods.

Reply:
I did the minimum changes so that the tests are passing. However a bunch of methods should need the 'pub' modifier. I did investigate and in fact I was no parsing the statements when looking for function calls.
Most of the stdlib is now 'pub'